src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -462,3 +462,13 @@ def test_torchvision_nms(pyi_builder):
         # is discarded.
         assert keep == 1
     """)
+
+
+@importorskip('googleapiclient')
+def test_googleapiclient(pyi_builder):
+    pyi_builder.test_source("""
+        from googleapiclient.discovery import build
+        # Actual access to the service requires API key...
+        #service = build('drive', 'v3')
+        #service.close()
```

Comment:
Are these meant to be commented out?

Reply:
Yes, because they need the API key to work properly.

Reply:
But in the hindsight, there's no point in having that unused code around, so it's gone with the latest commit.

Reply:
Ahh, I see it said so just above - I just didn't see it.